src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -233,6 +246,11 @@ public void setPreviousPositions(List<PersonPositionHistory> previousPositions)
     this.previousPositions = previousPositions;
   }
 
+  @GraphQLInputField(name = "allPreviousPositions")
+  public void setAllPreviousPositions(List<PersonPositionHistory> previousPositions) {
+    this.previousPositions = previousPositions;
+  }
+
```

Comment:
This field should not be settable through GraphQL (query-only field, if it should be there at all).

Reply:
GraphQLInput removed.